src/makeRobot.js

```diff
@@ -37,8 +37,67 @@
  *
  * @return {Robot}
  */
+
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      // eslint-disable-next-line max-len
+      return `name: ${this.name}, chip version: ${this.version}, wheels: ${this.wheels}`;
```

Comment:
try to use wrapping instead use `eslint-disable`
```suggestion
      return `name: ${
        this.name
      }, chip version: ${
        this.version
      }, wheels: ${
        this.wheels
      }`;
```

OR 

```suggestion
      const { name, version, wheels } = this;
      
      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
```

OR

```suggestion
      return `name: ${this.name}` + 
        `chip version: ${this.version}` +
        `wheels: ${this.wheels}`;
```